config/example/config.txt

```diff
@@ -282,33 +283,34 @@ USEALIENWHITELIST
 #ADMIN_IRC #admin
 
 ## Uncommen to allow ghosts to write in blood during Cult rounds.
-ALLOW_CULT_GHOSTWRITER
+CULT_GHOSTWRITER
 
 ## Sets the minimum number of cultists needed for ghosts to write in blood.
-REQ_CULT_GHOSTWRITER 6
+CULT_GHOSTWRITER_REQ_CULTISTS 6
 
 ## Sets the number of available character slots
 CHARACTER_SLOTS 10
 
-## Sets the number of loadout slots per character
-LOADOUT_SLOTS 3
-
 ## Expected round length in minutes
 EXPECTED_ROUND_LENGTH 180
 
 ## The lower delay between events in minutes.
 ## Affect mundane, moderate, and major events respectively
-EVENT_DELAY_LOWER 10;30;50
+EVENT_DELAY_LOWER 1 10 

```

Comment:
I need these event variables moved out of config entirely, make them hardcoded defines instead